protocol/README.md

```diff
@@ -15,6 +15,12 @@ The lower-level `Handshake` and `PacketHandler` types can be directly used by ap
 * `async` -- High level wrappers for asynchronous read and write runtimes using agnostic futures-rs traits.
 * `tokio` -- Same wrappers as `async`, but using the popular tokio runtime's specific traits instead of futures-rs.
 
+## Minimum Supported Rust Version (MSRV)
+
+This crate has a baseline MSRV of **1.63.0**.
+
+However, when using the `tokio` feature flag, the effective MSRV may be higher depending on which version of tokio is selected by the caller.
```

Comment:
I'd even say that any external dependency may possibly do this. `tokio` is currently the only one that actually does but it may happen if they backport MSRV bump to `rand` for instance.

Reply:
It's not that bad if they set `rust-version` and let a new `cargo` generate the lock file (they can run it in a VM or whatever, no risk there).

Reply:
Yea, I think the v3 resolver will help out here.

Reply:
IIRC it's unrelated to resolver version, just `cargo` being new enough but I might be wrong.

Reply:
Interesting!